superset/translations/fr/LC_MESSAGES/messages.json

```diff
@@ -4591,18 +4618,10 @@
       "y: values are normalized within each row": [""],
       "year": ["année"],
       "zoom area": [""],
-      "No matching records found": ["Aucun résultat trouvé"],
```

Comment:
Did you replace this string?  I see it deleted here but no other mention of "no matching records found"

Reply:
Just annotation fuzzy on this string, I remove it

Reply:
I don't follow, sorry.  Will the string "No matching records found" still be translated?